CoroutineTemplate/app/build.gradle.kts

```diff
@@ -15,13 +15,20 @@ val keystoreProperties = rootDir.loadGradleProperties("signing.properties")
 
 android {
     signingConfigs {
-        create("config") {
+        create(BuildType.RELEASE) {
             // Remember to edit signing.properties to have the correct info for release build.
-            storeFile = file("../keystore_production")
+            storeFile = file("../config/release.keystore")
             storePassword = keystoreProperties.getProperty("KEYSTORE_PASSWORD") as String
             keyPassword = keystoreProperties.getProperty("KEY_PASSWORD") as String
             keyAlias = keystoreProperties.getProperty("KEY_ALIAS") as String
         }
+
+        getByName(BuildType.DEBUG) {
```

Comment:
Interesting, I see `create(...)` & `getByName(...)`. Any idea why `RELEASE` & `DEBUG` are using different keywords? 🙏

Reply:
@Tuubz that's a magic of Kotlin DSL, I can't find an official doc yet, it's still new 🤪 https://gist.github.com/mileskrell/7074c10cb3298a2c9d75e733be7061c2